src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -1168,8 +1168,8 @@ protected function calculateNewTableColumns(string $tableName, string|false $col
             $writeColumns[] = $writeName;
         }
 
-        $selectColumns = array_filter($selectColumns, 'strlen');
-        $writeColumns = array_filter($writeColumns, 'strlen');
+        $selectColumns = array_filter($selectColumns, fn($v) => $v && strlen($v) > 0);
+        $writeColumns = array_filter($writeColumns, fn($v) => $v && strlen($v) > 0);
```

Comment:
Per https://github.com/phpstan/phpstan/issues/2266, the maintainer of phpstan made a conscious decision to encode this where the callable passed to `array_filter` must return a boolean value (I think), which strlen doesn't do.

Passing `'strlen'` would also implicitly filter out both emtpy strings and `false` values, whereas doing it explicitly throws a type error passing a boolean to `strlen`, so this is certainly more type safe in general I guess.